EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
Of course, but then in the output we would repeat the sequence again, so that 3 sequences are written. And 3 proteins.

Sure, one of the sequences is repeating, but this way there is no confusion which of the sequences corresponds to 2 proteins and which to 1.